{{cookiecutter.project_slug}}/environment-rtd.yml

```diff
@@ -0,0 +1,14 @@
+name: {{ cookiecutter.project_slug }}
```

Comment:
I would call this `environment-rtd.yml ` `environment-docs.yml` since Raven (https://github.com/Ouranosinc/raven/blob/8798356b324339fc13e16a95878d8453615ececb/environment-docs.yml), Finch (https://github.com/bird-house/finch/blob/cdadcd67f0ab2fb0811ce425f45501adb5bc5b97/environment-docs.yml) and Flyingpigeon (https://github.com/bird-house/flyingpigeon/blob/9f12aa319c4f190715c22b6cdf6cacdd4290b234/environment-docs.yml) are all named like that.